src/qt/combobox.cpp

```diff
@@ -36,6 +38,30 @@ wxQtComboBox::wxQtComboBox( wxWindow *parent, wxComboBox *handler )
             this, &wxQtComboBox::editTextChanged);
 }
 
+void wxQtComboBox::showPopup()
+{
+    wxComboBox *handler = GetHandler();
+    if( handler )
+    {
+        wxCommandEvent event( wxEVT_COMBOBOX_DROPDOWN, handler->GetId() );
+        event.SetEventObject( handler );
```

Comment:
AFAICS this is already done by `EmitEvent()`

Reply:
@vadz,
I'll check and if it does fix it.